README.adoc

```diff
@@ -178,60 +185,63 @@ The [hotspot=15-18 file=2]`handles()` method inspects the HTTP response code to
+First, you need to define the base URL of the [hotspot=SystemClient file=1]`SystemClient` instance.
 Configure the default base URL with the MicroProfile Config feature. This feature is enabled for you in both `server.xml` and `pom.xml` files.
 
-[role="code_command hotspot", subs="quotes"]
+[role="code_command hotspot file=0", subs="quotes"]
 ----
 #Create the configuration file.#
 `src/main/webapp/META-INF/microprofile-config.properties`
 ----
 microprofile-config.properties
-[source, properties, linenums, role='code_column']
+[source, properties, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/webapp/META-INF/microprofile-config.properties[tags=**;!copyright]
+include::finish/src/main/webapp/META-INF/microprofile-config.properties[]
 ----
 
-The config property [hotspot=1 file=0]`io.openliberty.guides.inventory.client.SystemClient/mp-rest/url` is configured to the default [hotspot=2 file=0]`\http://localhost:9080/system` base URL.
+The [hotspot=baseUri file=0]`mp-rest/uri` base URL config property is configured to the default [hotspot=baseUri file=0]`\http://localhost:9080/system` URL.
 
 This configuration is automatically picked up by the MicroProfile Config API.
 
-Look at the annotations in the [hotspot=17-22 file=1]`SystemClient` interface again.
+Look at the annotations in the [hotspot=SystemClient file=1]`SystemClient` interface again.
 
 SystemClient.java
-[source, Java, linenums, role='code_column']
+[source, Java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[tags=**;!copyright]
+include::finish/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[]
 ----
 
-The [hotspot=13 file=1]`@Dependent` annotation tells the service that the scope of the interface depends on the class that it is injected into.
-By default, interfaces have a `@Dependent` scope unless another scope is defined on the interface.
+The [hotspot=RegisterRestClient file=1]`@RegisterRestClient` annotation registers the interface as a RESTful client. The runtime creates a CDI managed bean for every interface that is annotated with the `@RegisterRestClient` annotation.
 
-The [hotspot=14 file=1]`@RegisterRestClient` annotation registers the interface as a RESTful client. The runtime creates a CDI managed bean for every interface that is annotated with the `@RegisterRestClient` annotation.
+The [hotspot=RegisterRestClient file=1]`configKey` value in the [hotspot=RegisterRestClient file=1]`@RegisterRestClient` annotation replaces the fully-qualified classname of the properties in the [hotspot file=0]`microprofile-config.properties` configuration file.
+For example, the `<fully-qualified classname>/mp-rest/uri` property becomes `systemClient/mp-rest/uri`.
+The benefit of using Config Keys is when multiple client interfaces have the same `configKey` value, the interfaces can be configured with a single MP config property.
```

Comment:
change
The benefit of using Config Keys is when multiple client interfaces have the same `configKey` value, the interfaces can be configured with a single MP config property.
TO
**The benefit of using Config Keys the interfaces to be configured with a single MP config property when multiple client interfaces have the same `configKey` value,**

Reply:
The original sentence is clearer.